app/assets/stylesheets/search.scss

```diff
@@ -0,0 +1,27 @@
+#post-container{
+  width: 100%;
+  background: white;
+  h6{
```

Comment:
Trukst atstrapes pēc h6!

Un vel, lieta tāda kā kad tu izmanto standartus html elementus priekš tekstam, viņu vajadzēs stingri nostīlot ar izveletu fontu un izmēru. Tāpēc kā katram pārlūkam patīk to radīt savā noteiktā veidā.

Pagaidām, kamēr tev vel nav sacies uzdevums ar nostilošanu, tu vari pielikt reset.css.
Viņš noņems visadus liekus stilus priekš html elementiem. Vari to uzlīkt ar yarn